features/signal/activities/README.md

```diff
@@ -0,0 +1,8 @@
+# Run activities from within a Workflow Signal
+
+Workflows can run activities from within a Signal handler.
+
+# Detailed spec
+
+Workflow Signal handlers are like workflow code in that they can invoke and wait on activities.
+This test invokes a number of activities within the Signal and blocks on them, returning from the Signal when they have all completed.
```

Comment:
Is this a feature worth having? I created it to test my own understanding, and I couldn't think of a reason why this feature should exist for Update and not Signal and Query (which someone can add later).

Reply:
Sure, worth having IMO. Anything that is a clear callout to something we need to support can be a feature.